src/Utilities/Blaze/StepFunction.hpp

```diff
@@ -0,0 +1,76 @@
+// Distributed under the MIT License.
```

Comment:
Would it make more sense to move this into `DataStructures/Blaze`? It's not like anything in this will be used elsewhere and having a `blaze` spectre library might be a bit odd. Having said that, I think the two possibilities I like most are:
1. move to `DataStructures/Blaze`
2. add the headers to the blaze target instead of `Utilities`

Reply:
`DataStructures/Blaze` sounds good 👍 changed the first commit.